deploy/alerts-preprod.yaml

```diff
@@ -26,11 +26,11 @@ spec:
       action: "Sjekk app {{ $labels.app }} i namespace {{ $labels.kubernetes_namespace }} sine selftest for å se hva som er galt"
     - alert: Høy andel HTTP serverfeil (5xx responser)
       severity: danger
-      expr: (100 * (sum by (backend) (rate(traefik_backend_requests_total{code=~"^5\\d\\d", backend=~"https://medlemskap-oppslag.nais.preprod.local"}[3m])) / sum by (backend) (rate(traefik_backend_requests_total{backend=~"https://medlemskap-oppslag.nais.preprod.local"}[3m])))) > 1
+      expr: (100 * (sum by (backend) (rate(traefik_backend_requests_total{code=~"^5\\d\\d", backend=~"medlemskap-oppslag.nais.preprod.local.*"}[3m])) / sum by (backend) (rate(traefik_backend_requests_total{backend=~"medlemskap-oppslag.nais.preprod.local.*"}[3m])))) > 1
       for: 3m
       action: "Sjekk loggene for å se hvorfor {{ $labels.backend }} returnerer HTTP feilresponser"
     - alert: Høy andel HTTP klientfeil (4xx responser)
       severity: warning
-      expr: (100 * (sum by (backend) (rate(traefik_backend_requests_total{code=~"^4\\d\\d", backend=~"https://medlemskap-oppslag.nais.preprod.local"}[3m])) / sum by (backend) (rate(traefik_backend_requests_total{backend=~"https://medlemskap-oppslag.nais.preprod.local"}[3m])))) > 10
+      expr: (100 * (sum by (backend) (rate(traefik_backend_requests_total{code=~"^4\\d\\d", backend=~"medlemskap-oppslag.nais.preprod.local.*"}[3m])) / sum by (backend) (rate(traefik_backend_requests_total{backend=~"medlemskap-oppslag.nais.preprod.local.*"}[3m])))) > 10
```

Comment:
Samme her, fjern punktum. Gjør det samme i alerts-prod

Reply:
Dette vil også være mulig å debugge/dobbeltsjekke med explore i Grafana, for å se om du får verdi fra traefik_backend_requests_total. Kan anbefales å leke litt med spørringer og se hva som skjer hvis du endrer verdier etc